### PR TITLE
Fix skipped status text color

### PR DESCRIPTION
**Pull request by bisht-richa**
## What does this PR change?
- Fixed skipped tasks and read messages color contrast in table

**add description**

## GUI diff

No difference.

**Before:**

Screenshot 2024-04-25 at 16 08 39

Screenshot 2024-04-25 at 16 11 27

**After:**

Screenshot 2024-04-25 at 16 09 03

Screenshot 2024-04-25 at 16 09 33

- [x] **DONE**

## Documentation
- No documentation needed

- [ ] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Issue(s): #[#24136](https://github.com/SUSE/spacewalk/issues/24136)
Port(s): # **add downstream PR(s), if any**

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Comment by rjmateus** [comment]
Maybe instead of having the text in a different color as before (which makes it hard to read), maybe we have have a full row with a different (maybe gray) color?

**Comment by bisht-richa** [comment]
> While this fixes the contrast issue, the two types of entries are now practically indistinguishable, e.g. I can't tell that there's any difference between the rows in the after image. Is this intentional? If so, we might want to use some kind of a different indicator for these statuses.

We already have `open` and 'closed` envelop icons to show the two types of entries on the Notification page. And in another table, we have a clear indicator of `Skipped` status. 
I don't know if we need more indicators here.

**Comment by Etheryte** [comment]
I don't have a strong opinion on how we indicate the status, however if we want to remove the muted text visual here, it would be more appropriate to remove the class from the elements, rather than override the style for it. Right now we override a style back to the base it already was.

File: web/html/src/branding/css/susemanager/components/text.less [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// These are all leftovers from the old theme, hopefully we can drop most of this over time
+.text-muted {
```
Comment (Etheryte) [inline web/html/src/branding/css/susemanager/components/text.less:2]: Do we still need this if we're removing the class from the DOM? Same for the scss one.

Comment (bisht-richa) [reply web/html/src/branding/css/susemanager/components/text.less:2]: Yes, we still have use cases where we are using this class from scss file. But I will remove this text.less file I think we don't need it.

**Review by Etheryte — APPROVED** [review]

**Pull request merged**